Formula/pdns.rb

```diff
@@ -61,19 +64,23 @@ def plist; <<-EOS.undent
     <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
     <plist version="1.0">
     <dict>
-      <key>KeepAlive</key>
-      <true/>
       <key>Label</key>
       <string>#{plist_name}</string>
       <key>ProgramArguments</key>
       <array>
-        <string>#{opt_bin}/pdns_server</string>
+        <string>#{sbin}/pdns_server</string>
       </array>
-      <key>EnvironmentVariables</key>
-      <key>KeepAlive</key>
+      <key>RunAtLoad</key>
       <true/>
-      <key>SHAuthorizationRight</key>
-      <string>system.preferences</string>
+      <key>KeepAlive</key>
+      <dict>
+        <key>Crashed</key>
+        <true/>
+      </dict>
+      <key>StandardErrorPath</key>
+      <string>#{var}/log/#{name}/pdns_server.err</string>
+      <key>StandardOutPath</key>
+      <string>/dev/null</string>
```

Comment:
Based on cursory research, nothing really seems to come from STDOUT besides the output from `--help`